### ISSUE TITLE
Build fails on the dongle

### ISSUE THREAD

**Issue by mondalaci** [UltimateHackingKeyboard/firmware#916]
```
/home/laci/projects/firmware-uhk80/uhk/device/src/legacy/event_scheduler.c:78: undefined reference to `Uart_Enable'
```

**Comment by kareltucek** [comment]
Please, add relevant stub to legacy/stubs.h, or disable it via an ifdef

**Comment by mondalaci** [comment]
Thanks! Fixed by eaac9112bbb4f6022606f9b929d9bb18f10dc45a
